src/deluge/gui/views/session_view.cpp

```diff
@@ -920,7 +921,7 @@ ActionResult SessionView::padAction(int32_t xDisplay, int32_t yDisplay, int32_t
 
 				// If Clip is present here
 				if (clip) {
-
+					currentSong->currentClip = clip;
```

Comment:
I don't know that this should select clips - remember the purpose of currentClip is to control the target for looper footswitches and midi follow. Muting/launching a clip might not be a great indication that the user wants to select it